### PR TITLE
Unused arguments deleted

### LINKED ISSUE #13: Delete unused arguments
After some refactoring operations, a couple of method arguments in the module `populate_db.py` seem to be no longer needed.

In particular, I refer to:

- `is_ready_for_db` of the `load_table` method: https://github.com/collab-uniba/KGTorrent/blob/1df4a1ed9258f2214856da2129723a3e23d17c1a/KGTorrent/populate_db.py#L123
- `df` of the `write_table` method: https://github.com/collab-uniba/KGTorrent/blob/1df4a1ed9258f2214856da2129723a3e23d17c1a/KGTorrent/populate_db.py#L191

They should be removed.

### PR DESCRIPTION
**Pull request by Giovalo**
I delete unused arguments

close #13

### CODE REVIEW

File: KGTorrent/populate_db.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def check_table_emptiness(table_name, engine):
             return False
         else:
             return True
-
+    """
```
Comment (louieQ) [inline KGTorrent/populate_db.py:38]: Hi Giovanni, here it seems to me that you misused the multi-line string syntax. Multi-line strings in Python are marked by triple quotes `"""`; when they are not used in the context of assignments, they usually serve as multi-line docstrings. On the other hand, they are not meant to be used as multi-line comments, although they are functionally equivalent to them, being ignored by Python interpreters.

PEP8 suggests using dashes `#` for both single-line and multi-line comments and in this project we stick to the PEP8 coding standard. PyCharm is PEP8 compliant too: toggling comments in multi-line selections results in prepending lines with dashes.

---

By the way, in this specific context, I encourage you to delete this function if it is no longer used. Should it turn useful again in the future, we would still be able to restore it from former commits.

**Review by louieQ — COMMENTED** [review]
The changes in this PR seem OK to me. Have you tried to re-run the scripts to ensure that they are still working properly?

File: KGTorrent/populate_db.py (hunk 1 above)
Comment (Giovalo) [reply KGTorrent/populate_db.py:38]: I'm really sorry Luigi, i forgot to tidy up triple quotes. That was a quick check to see if it is an unused function. As you sudgest i'm going to remove it and i'll be using `#` from now in order to comply with the standards.

**Comment by louieQ** [comment]
Ok @Giovalo, everything seems OK to me now. Feel free to merge this branch to `master`.

**Pull request merged**